File: BACKEND/PickleballCourtBookingSystem/PickleballCourtBookingSystem.Core/Entities/CourtCluster.cs
namespace PickleballCourtBookingSystem.Core.Entities

{
    public class CourtCluster
    {
        public Guid? Id { get; set; }
        public string? Name { get; set; }
        public string? Description { get; set; }
        public TimeSpan? OpeningTime { get; set; }
        public TimeSpan? ClosingTime { get; set; }
        public Guid? AddressId { get; set; }
        public Guid? CourtOwnerId { get; set; } 
        public int? Status { get; set; }
    }
}

File: AndroidApp/app/src/main/java/com/maxholmes/androidapp/data/model/CourtCluster.kt
package com.maxholmes.androidapp.data.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import kotlin.time.Duration

@Parcelize
data class CourtCluster(
    var id: String,
    var name: String,
    var openingTime: String,
    var closingTime: String,
    var description: String?,
    var address: Address,
    var courtOwnerId: String,
    var imageUrl: String? = null,
    var status: String
) : Parcelable
